src/infoPanel/infoPanel.c

```diff
@@ -197,6 +197,7 @@ int main(int argc, char *argv[])
 				}
 			}
 		}
+		msleep(100);
```

Comment:
100ms ruins the experience! Maximum 15ms, I recommend 4ms.

Reply:
done